cluster/cluster_service.proto

```diff
@@ -41,6 +43,9 @@ service TypeDBCluster {
     rpc user_password (ClusterUser.Password.Req) returns (ClusterUser.Password.Res);
     rpc user_delete (ClusterUser.Delete.Req) returns (ClusterUser.Delete.Res);
 
+    // User Token API
+    rpc user_token_renew (ClusterUserToken.Renew.Req) returns (ClusterUserToken.Renew.Res);
```

Comment:
Should this perhaps be `SessionToken`?

Reply:
A "user token" isn't only used to authenticate session. For instance, it is also used to authenticate database management operations.

Given that one user will have at most one token per server instance, I think the name user token is apt.